rest-server/bin/harness-start

```diff
@@ -13,10 +13,10 @@ HARNESS_CLASS=com.actionml.HarnessServer
 HARNESS_OPTS="\
 -Dlogback.configurationFile=${HARNESS_LOG_CONFIG} \
 -DHARNESS_LOG_PATH=${HARNESS_LOG_PATH} \
--Drest-server.host=${REST_SERVER_HOST} \
--Drest-server.port=${REST_SERVER_PORT} \
--Dmongo.host=${MONGO_HOST} \
--Dmongo.port=${MONGO_PORT}"
+-Drest-server.host=${HARNESS_HOST} \ # is this needed?
+-Drest-server.port=${HARNESS_PORT} \ # is this needed? We use a URI here
+-Dmongo.host=${MONGO_HOST} \ # is this needed?
+-Dmongo.port=${MONGO_PORT}" # is this needed? We use a URI here
```

Comment:
@andreyshikov are these really used? If so why do they have no effect on our switch to using the Mongo URI? I assume these actually set the values of akka conf values like `rest-server.host` and `mongo.port`

I also assume `mongo,host` is never used anymore. 

This is yet another reason we should completely remove akka conf. We must use the env for containers, this just makes problems like unused launch options more difficult to see.

Reply:
I believe these -D for config are not needed, is this correct @andreyshikov ?